src/modules/admin/navigations/types.ts

```diff
@@ -0,0 +1,42 @@
+export type ContentPickerType =
+	| 'CONTENT_PAGE'
+	| 'COLLECTION'
```

Comment:
Collections in hetarchief will not have their own url, they will always point to a (media) object directly, so we will use the 'ITEM' option and not 'COLLECTION'
```suggestion
```